bin/sh/sh.1

```diff
@@ -2921,7 +2922,9 @@ This version of
 was rewritten in 1989 under the
```

Comment:
```suggestion
, the Almquist shell,
was rewritten in 1989 under the
```

Reply:
Another idea, but maybe bloat. The comma would have to go on the line above, but github won't let me suggest that.